src/main/webapp/jsp/studentCourseDetails.jsp

```diff
@@ -9,7 +9,7 @@
     <script type="text/javascript" src="/js/student.js"></script>
 </c:set>
 
-<ts:studentPage bodyTitle="Team Details for ${data.studentCourseDetailsPanel.courseId}" pageTitle="TEAMMATES - Student">
+<ts:studentPage bodyTitle="Team Details for ${data.studentCourseDetailsPanel.courseId}" pageTitle="TEAMMATES - Student" jsIncludes="${jsIncludes}">
```

Comment:
I can remove it here and it won't affect anything (including it won't affect anything either) but in the near future where I'm going to tidy up the js codes I'm going to include it, anyway.